server/test/routes/article.test.js

```diff
@@ -0,0 +1,375 @@
+/* eslint-disable */
+import chai from 'chai'
+import chaiHttp from 'chai-http'
+import server from '../../index';
+import tokenService from '../../utils/services/tokenService';
+import mockData from './mockData';
+
+const testUser = mockData.user7; 
+
+const { expect } = chai;
+const baseUrl = '/api/articles/';
+const SIGN_UP_ROUTE = '/api/users';
+chai.use(chaiHttp);
+let slug;
+const articleTest = () => {
+describe('Testing articles routes', () => {
+    let token = tokenService.generateToken(testUser, '3d');
+      describe('Creating an Article', () => {
+        it('Should create an Article sucessfully with title, description and/or body specified', (done) => {
+            chai.request(server)
+            .post(`${baseUrl}`)
+            .send({
+                article:{
+                    title: 'A new title',
+                    description: 'A new description',
+                    body: 'A new body'
+                }
+            })
+            .set('Authorization', token)
+            .end((err, res) => {
+                expect(res.status).to.equal(201);
+                expect(res.body).to.have.property('article');
```

Comment:
Your assertions are not complete, you need to test the payload of the response to make sure it is returning what you expect.